src/services/user.service.js

```diff
@@ -0,0 +1,67 @@
+import models from '../models';
+
+const { Users } = models;
+
+/**
+ * @exports
+ * @class UserService
+ */
+class UserService {
+  /**
+ * create new user
+ * @static
+ * @param {object} newUser
+ * @memberof UserService
+ * @returns {object} data
+ */
+  static addUser(newUser) {
+    return Users.create(newUser);
+  }
+
+  /**
+ * find user by email
+ * @static
+ * @param {object} userEmail
+ * @memberof UserService
+ * @returns {object} data
+ */
+  static findByEmail(userEmail) {
+    return Users.findOne({
+      where: {
+        email: userEmail
+      },
+      logging: false
+    });
+  }
+
+  /**
+   *
+   *
+   * @static
+   * @param {item} user
+   * @param {value} token
+   * @returns {updated} @memberof UserService
+   */
+  static updateToken(user, token) {
+    return Users.update({ token }, {
+      where: {
+        id: user.id
+      },
+      logging: false
+    });
+  }
+
+  /**
+   *
+   * @param {string} token it acepts a valid token
+   * @returns {boolean} returns true when token is found otherwise false
+   */
+  static findByToken({ token }) {
+    return Users.findOne({
+      where: { token },
+      logging: false
```

Comment:
There is no need to put this property here since it has already been disabled in the config